src/Microsoft.OpenApi/Models/OpenApiParameter.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.OpenApi.Models
     public class OpenApiParameter : IOpenApiSerializable, IOpenApiReferenceable, IEffective<OpenApiParameter>, IOpenApiExtensible
     {
         private bool? _explode;
-        public ParameterStyle? _style;
+        private ParameterStyle? _style;
```

Comment:
this is a breaking change, the PR should be rebased on v2

Reply:
Done.

Reply:
I think something went wrong with the rebase, it's bringing way more commits than it should. If we need to refresh the v2 branch first, let's do it as a separate PR if you don't mind. Let me know if you need help